### PR TITLE
Carlos introduction

### PR DESCRIPTION
**Pull request by carlosespino11**

### CODE REVIEW

File: _posts/2016-06-23-carlos-intro.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Introducing DSSG Fellow Carlos Espino"
+date:   2016-06-23
+author: Carlos Espino
+---
+<p>
+I am a graduate student of Data Science at Columbia University. I previously worked for 3 years at Sinnia ([link](http://www.sinnia.com/en/index.html)) , a startup dedicated to social network analysis, as a Data Scientist on data visualization, text mining, machine learning and statistical modeling. I was born and raised in Mexico City, where I received my Bachelor's degree in Applied Mathematics at ITAM (Mexico Autonomous Institute of Technology). During my undergraduate studies I became passionate about statistics, mathematics and information technology, which led me to the field of Data Science. Growing up, I realized that the socioeconomic inequality prevalent in my home country has yet to be adequately addressed. I feel that as an advanced Data scientist and statistician, I can achieve more for my country and its inhabitants by applying what I learn in this fellowship back home for the betterment of the Mexican community.
```
Comment (arokem) [inline _posts/2016-06-23-carlos-intro.md]: No need for the space before the comma in the second sentence.

File: _posts/2016-06-23-carlos-intro.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+</p>
+<p>
+This field has the potential to have a major, positive impact on public policy and social services, and the DSSG program provides an environment for its fellows to combine social and computational sciences and develop real world solutions for social and urban development problems. This intersection of expertise will follow me in my career and provide me the skills and wherewithal to make a positive impact in other avenues. 
+</p>
+
+<p>
+The first week I met the other fellows and got to know my team, and I was a helper in the R Software Carpentry workshop. I am excited about spending the summer in Seattle, and I expect to go kayaking, play basketball and soccer, go to the CenturyLink field and hike. It´s gonna be a great summer...Olé 🇲🇽!
```
Comment (arokem) [inline _posts/2016-06-23-carlos-intro.md:15]: Do you know whether that emoji will work well in html?

File: _posts/2016-06-17-intro.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (arokem) [inline _posts/2016-06-17-intro.md]: Did you mean to remove this file?

Comment (carlosespino11) [inline _posts/2016-06-17-intro.md]: I think I committed it accidentally. I'll remove it.

File: _posts/2016-06-23-carlos-intro.md (hunk 2 above)
Comment (carlosespino11) [inline _posts/2016-06-23-carlos-intro.md:15]: I think it'll work.

**Comment by arokem** [comment]
⚽

**Pull request merged**